toolchain/c-toolchain.lisp

```diff
@@ -57,18 +57,25 @@
 
 ;;; C support
 
-(defparameter *cc*
-  #+(or cygwin (not windows)) "cc"
-  #+(and windows (not cygwin)) "gcc")
+(defparameter *default-cc*
+  ;; the copy-seq ensures we have a fresh string
+  ;; which allows for safe testing with EQ for someone else
+  ;; having set it.
+  (copy-seq
+   (or (getenv "CC")
+       #+(or cygwin (not windows)) "cc"
+       #+(and windows (not cygwin)) "gcc")))
 
-(defparameter *cc-flags*
-  (append
-   ;; For MacPorts
-   #+darwin (list "-I" "/opt/local/include/")
-   ;; ECL internal flags
-   #+ecl (parse-command-flags c::*cc-flags*)
-   ;; FreeBSD non-base header files
-   #+freebsd (list "-I" "/usr/local/include/")))
+(defparameter *cc* *default-cc*)
```

Comment:
I see.I suggest you don't worry about that, because the c-toolchain system is quite new and the only documented way of interacting with these was through the grovel directives. 

It's just that storing environment info from build time seems random and potentially confusing. (You've described this behaviour before, but I didn't fully grasp it then.) 

What do you think?